README.md

```diff
@@ -185,6 +182,36 @@ done
 
 The folder `~/historical-workspace/db` is now ready to be copied to the observer's working directory.
 
+## Storage Pruning
+
+At this moment, we only support a [storage pruning](https://www.rosetta-api.org/docs/storage_pruning.html#docsNav) mechanism that is **manually triggered** - the Observer behind Rosetta does not perform any _automatic pruning_. Instead, removing old epochs should be performed as follows:
+ - stop Observer and Rosetta
+ - remove epochs `[oldest, ..., latest - N - s]` (where `N` is the desired number of historical epochs, and `s = 2`, see below) from `/db/{chainID}` (or `/data/{chainID}` when using the Docker setup)
+ - start Observer and Rosetta
+
+The constant `s = 2` is needed to overcome possible snapshotting-related edge-cases, still present as of September 2022 (the use of this constant will become obsolete in future releases).
+
+### **Pruning example**
+
+For example, let's say that the `/db/{chainID}` (or `/data/{chainID}`) folder contains:
+ - the folder `Static`
+ - the folders `Epoch_1000, Epoch_1001, ..., Epoch_2022`.
```

Comment:
perhaps different numbers? so they cannot be seen as years

Reply:
Fixed.